src/org/jetbrains/java/decompiler/main/ClassWriter.java

```diff
@@ -204,6 +205,8 @@ public void classLambdaToJava(ClassNode node, TextBuffer buffer, Exprent method_
           }
           buffer.append(" ->");
 
+          // @martrixx: left here!
```

Comment:
Debug comment left over?

Reply:
Oh, this is just for me to keep track of from where I should continue working. I'll remove it in the next commit (and may add another one somewhere else, if I don't finish)